grpc/server/src/chronicles.rs

```diff
@@ -329,6 +353,25 @@ fn read_chronicle_template(c: Container, action: ChronicleAs, context: &mut Ctx)
         name.push(arg.into());
```

Comment:
@arbimo `at line 351` The action parameters cannot be converted into `SVar`.

Produces,
```
thread 'main' panicked at 'Variable with empty symbolic domain (note that we do not properly handle optionality in this case)', aries/model/src/model.rs:177:13
```

The file is at [model.rs](https://github.com/franklinselva/aries/blob/protobuf-interface/model/src/model.rs)